### ISSUE TITLE
Give method to clear db

### ISSUE THREAD

**Issue by fan-tom** [amplitude/Amplitude-iOS#258]
I incorrectly used `logEvent(..., withTimestamp: Date().timeIntervalSince1970)`, providing fractional number instead of integer. Event was saved to local db, but then I cannot send it (and all subsequent events too) to backend, because `ERROR: Connection response received:400, invalid_event_field. Invalid fields [timestamp].`. I need some way to clear db

**Comment by haoliu-amp** [comment]
Are you trying to clear the db on one developer's device?

**Comment by fan-tom** [comment]
> Are you trying to clear the db on one developer's device?
yes

**Comment by haoliu-amp** [comment]
If it's just from your personal developer's device, delete the app and reinstall it would solve it. Not sure if I misunderstand it.

**Comment by fan-tom** [comment]
Yeah, sure, I can reinstall app or even remove file on my dev device/emulator. But what if I already released application and then realized error. I cannot issue update that will simply clear db on application start. How can I fix it on client's device? Or maybe we doesn't need such a method if we guarantee that any event, written to db will eventually be successfully sent to backend or discarded, so no improperly created event will prevent subsequent events from being sent to backend. Currently, they are stay at db, even if they cannot be accepted by backend without modification (as in my case with invalid timestamp)

**Comment by haoliu-amp** [comment]
Make sense, we will look into it.
